authors/apps/articles/views.py

```diff
@@ -1,29 +1,15 @@
-
 from django.db.models import Avg
+from .models import Article, Ratings
+from django.db.models import Count
 from rest_framework import mixins, status, viewsets,generics
-from rest_framework.permissions import IsAuthenticatedOrReadOnly
 from rest_framework.exceptions import NotFound, PermissionDenied
 from rest_framework.permissions import IsAuthenticatedOrReadOnly, AllowAny
-from rest_framework.exceptions import NotFound, PermissionDenied
-
-from .serializers import ArticleSerializer, RatingSerializer, TagSerializer, CommentSerializer
-from rest_framework.response import Response
 from rest_framework import mixins, status, viewsets, generics
 from rest_framework.views import APIView
-from rest_framework.pagination import PageNumberPagination
+from rest_framework.response import Response
 from .models import Article, Ratings, Comment, Tag
-from .serializers import ArticleSerializer, RatingSerializer
-from .renderers import ArticleJSONRenderer, RatingJSONRenderer,CommentJSONRenderer
-
-
-
-class LargeResultsSetPagination(PageNumberPagination):
-    """
-    Set pagination results settings
-    """
-    page_size = 10
-    page_size_query_param = 'page_size'
-    max_page_size = 10
```

Comment:
@dcDalin I need your eyes on this file, check if your pagination feature was not accidentally dropped